frontend/src/test.setup.ts

```
@@ -0,0 +1,15 @@
+import {Window, Document} from "happy-dom";
+
+declare const globalThis: {
```

Biome (2.1.2)

[error] Do not shadow the global "globalThis" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)